doc/LocalVMSetup.md

```diff
@@ -16,8 +16,18 @@ You can download CentOS 7.8.2003 ISO file from http://www.centos.org, or from an
 ## 3. Install a fresh CentOS 7.8.2003
 Automatic install or manual install. Basic installation is OK. When you build CORTX from source in next steps, dependant packages will be resolved and installed.
 
-## 4. You may need to add EPEL repo.
-Please refer to: https://fedoraproject.org/wiki/EPEL
+## 4. Run the following commands.
+
+ sudo yum install epel-release
+
+ sudo yum repolist
+
+ sudo yum install ansible
+
+ wget https://archives.fedoraproject.org/pub/archive/epel/5/x86_64/epel-release-5-4.noarch.rpm
+ sudo rpm –ivh epel-release-latest-5.noarch.rpm
+ sudo yum repolist
+ sudo yum install ansible
```

Comment:
These four lines are doing the same as previous three lines? 


ps. the format of these lines are broken.

Reply:
Yes I don't think these lines are needed, you are right